integration-tests/src/test/kotlin/com/google/devtools/ksp/test/PlaygroundIT.kt

```diff
@@ -59,7 +59,7 @@ class PlaygroundIT {
         val gradleRunner = GradleRunner.create().withProjectDir(project.root)
 
         File(project.root, "workload/build.gradle.kts")
-            .appendText("\nksp {\n  blockOtherCompilerPlugins = true\n}\n")
+            .appendText("\nksp {\n  blockOtherCompilerPlugins = false\n}\n")
```

Comment:
what is this for?

Reply:
I'd like to deprecate the flag gradually. The default becomes true effectively. Let's not break `blockOtherCompilerPlugins = false` too soon.